search.py

```diff
@@ -0,0 +1,22 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+
+import sys
+import louis.db.api as api
+import louis.db as db
+#from microbench import MicroBench
+#import pandas as pd
+
+#basic_bench = MicroBench()
+
+# Execute the SQL search function 
+#@basic_bench
```

Comment:
you can create a small wrapper function here to call out to the db module so you can still benchmark